src/classes/PPlan_Primary_TDTM.cls

```diff
@@ -80,13 +110,19 @@ public with sharing class PPlan_Primary_TDTM extends TDTM_Runnable {
                 }
             }
         }
+        
         if (acccountIdsNeedNonPrimarycount.size()>0) {
             dmlWrapper.objectsToUpdate.addAll(makeOtherPPlanNotPrimary(acccountIdsNeedNonPrimarycount, primaryProgramIds));
         }
+		
         if (triggerAction == TDTM_Runnable.Action.BeforeUpdate) {
             TDTM_ProcessControl.setRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_Before_Update, false);
         }else if (triggerAction == TDTM_Runnable.Action.BeforeInsert) {
             TDTM_ProcessControl.setRecursionFlag(TDTM_ProcessControl.registeredTrigger.PPlan_Primary_TDTM_Before_Insert, false);
+        }else if (triggerAction == TDTM_Runnable.Action.AfterUpdate) {
```

Comment:
I'm not going to have you change this as it looks like that style was there before, but want to call this to Carlos (@ceiroa): someone should perhaps verify that our else blocks don't really look like this. i think its hard to read with that indentation (maybe its just GitHub).